### PR TITLE
update IDE versions

### PR DESCRIPTION
**Pull request by pq**
/cc @stevemessick @devoncarew

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ A brief summary of the [getting started guide](https://flutter.io/docs/developme
 - install the [Flutter SDK](https://flutter.io/docs/get-started/install)
 - run `flutter doctor` from the command line to verify your installation
 - ensure you have a supported [IntelliJ development environment](https://www.jetbrains.com/idea/download), either:
-  - IntelliJ 2017.3 or 2018.1, Community or Ultimate Edition, or
-  - Android Studio 3.1 (note: Android Studio Canary versions are generally _not_ supported)
+  - IntelliJ 2017.3 or 2018.3, Community or Ultimate Edition, or
```
Comment (pq) [inline README.md]: should I drop 2017.3?

Comment (stevemessick) [reply README.md]: Yes, see product-matrix.json

Comment (devoncarew) [reply README.md]: From the product-matrix.json file, it looks like our lower bound is `2018.1`

Comment (pq) [reply README.md]: Ah.  Sure.  I should have looked there.  Thanks!

**Comment by pq** [comment]
Updated.

**Pull request merged**